docker/Dockerfile

```diff
@@ -24,47 +24,38 @@ RUN install_packages \
     libbz2-dev \
     liblzma-dev
 
-# install java-8    (adds 340Mb)
+# install java-11    (adds 340Mb)
 RUN install_packages default-jdk
 
-# install python3.7 (adds 800Mb)
+# install python3.10 (adds 800Mb)
 RUN apt-get update \
-	&& apt-get dist-upgrade -y \
-	&& apt-get install -y --no-install-recommends python-smbus libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev libssl-dev openssl libffi-dev \
-	&& apt-get clean \
+       && apt-get dist-upgrade -y \
+       && apt-get install -y --no-install-recommends python3-smbus python3-pkg-resources libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev libssl-dev openssl libffi-dev \
+       && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-ENV PYTHON_VERSION="3.7.1"
+ENV PYTHON_VERSION="3.10.13"
```

Comment:
I had some error [like this](https://stackoverflow.com/questions/74550830/error-could-not-build-wheels-for-aiohttp-which-is-required-to-install-pyprojec) while trying versions above python3.9. But it looks like its fixed in the meantime. 3.10 is okay.